File: app/src/main/java/com/babestudios/companieshouse/data/model/search/CompanySearchResultItem.java
package com.babestudios.companieshouse.data.model.search;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class CompanySearchResultItem {

	@SerializedName("companyLinks")
	public Links links;

	@SerializedName("description_identifier")
	public List<String> descriptionIdentifier = new ArrayList<String>();

	@SerializedName("date_of_creation")
	public String dateOfCreation;

	@SerializedName("snippet")
	public String snippet;

	@SerializedName("company_number")
	public String companyNumber;

	@SerializedName("title")
	public String title;

	@SerializedName("company_status")
	public String companyStatus;

	@SerializedName("matches")
	public Matches matches;

	@SerializedName("address")
	public Address address;

	@SerializedName("description")
	public String description;

	@SerializedName("kind")
	public String kind;

	@SerializedName("company_type")
	public String companyType;

	@SerializedName("address_snippet")
	public String addressSnippet;

}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/search/CompanySearchResultItem.kt
package com.babestudios.companyinfouk.data.model.search

import com.google.gson.annotations.SerializedName

import java.util.ArrayList

class CompanySearchResultItem {

	@SerializedName("companyLinks")
	var links: Links? = null

	@SerializedName("description_identifier")
	var descriptionIdentifier: List<String> = ArrayList()

	@SerializedName("date_of_creation")
	var dateOfCreation: String? = null

	@SerializedName("snippet")
	var snippet: String? = null

	@SerializedName("company_number")
	var companyNumber: String? = null

	@SerializedName("title")
	var title: String? = null

	@SerializedName("company_status")
	var companyStatus: String? = null

	@SerializedName("matches")
	var matches: Matches? = null

	@SerializedName("address")
	var address: Address? = null

	@SerializedName("description")
	var description: String? = null

	@SerializedName("kind")
	var kind: String? = null

	@SerializedName("company_type")
	var companyType: String? = null

	@SerializedName("address_snippet")
	var addressSnippet: String? = null

}
